core/domain/classifier_services.py

```diff
@@ -417,12 +433,35 @@ def _add_examples(self, examples, iterations):
     def add_examples_for_training(self, training_examples):
         """Adds examples to the classifier with _training_iterations number of
         iterations.
+
+        Args:
+            training_examples: list of examples. A single example is a doc
+                with a list of labels it should be matched to, e.g:
+
+                training_examples = [
+                    ['i eat fish and vegetables', ['food']],
+                    ['fish are pets', ['pets']],
+                    ['my kitten eats fish', ['food', 'pets']]
+                ]
+
+        Returns:
+            xrange. An iterator over the ids of the docs just added.
         """
         return self._add_examples(training_examples, self._training_iterations)
 
     def add_examples_for_predicting(self, prediction_examples):
         """Adds examples to the classifier with _prediction_iterations number
         of iterations.
+
+        Args:
+            prediction_examples: list of str. A list of docs.
```

Comment:
What do you think about renaming prediction_examples to prediction_docs throughout, and calling this method add_docs_for_predicting, in order to reduce confusion about what 'example' actually means? I think we should consistently use the term 'example' to refer to a doc-with-labels.